ci/ray_ci/tester.py

```diff
@@ -416,8 +417,33 @@ def _get_high_impact_test_targets(
         if test.get_oncall() == team
     }
     changed_tests = _get_changed_tests()
+    human_specified_tests = _get_human_specified_tests()
 
-    return high_impact_tests.union(changed_tests)
+    return high_impact_tests.union(changed_tests).union(human_specified_tests)
+
+
+def _get_human_specified_tests() -> Set[str]:
+    """
+    Get all test targets that are specified by humans
+    """
+    base = os.environ.get("BUILDKITE_PULL_REQUEST_BASE_BRANCH")
+    head = os.environ.get("BUILDKITE_COMMIT")
+    if not base or not head:
+        # if not in a PR, return an empty set
+        return set()
+
+    tests = set()
+    subprocess.check_call(["git", "fetch", "origin", base], cwd=bazel_workspace_dir)
+    messages = subprocess.check_output(
+        ["git", "rev-list", "--format=%B", f"origin/{base}...{head}"],
+        cwd=bazel_workspace_dir,
+    )
+    for message in messages.decode().splitlines():
+        if message.startswith(MICROCHECK_COMMAND):
+            tests = tests.union(message[len(MICROCHECK_COMMAND) :].strip().split(" "))
```

Comment:
this is decoding all messages? why not just the last one? I think we just need the last one?

is it possible to just check the git body, but not the title? then there is a lot less risks of having `@microcheck` polluting the title, and much easier to meet the %0.3 goal.

and I am much more tolerant to having unrelated info in the commit message body.

Reply:
intentionally all messages yes, so that folks don't need to re-add tests again on every commit update

let me check how to get the body vs title

Reply:
use `format=%b` to get the git body only